gitlab/resource_gitlab_user.go

```diff
@@ -92,6 +92,12 @@ func resourceGitlabUser() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"state": {
+				Description: "String, defaults to 'active'. The state of the user account.",
```

Comment:
The description should mention "blocked" as well. You could also add a field here - `ValidateFunc: validation.StringInSlice([]string{"active", "blocked"}, false)` - to validate values.

Reply:
Added both